cpp/src/aztec/honk/CMakeLists.txt

```diff
@@ -1 +1 @@
-barretenberg_module(honk numeric ecc srs plonk proof_system)
+barretenberg_module(honk numeric ecc srs plonk proof_system transcript)
```

Comment:
Why does honk depend on plonk?

Reply:
I just testing removing it and had no issue. I guess you have made this possible with this PR or the proof_system move PR. Would you remove plonk as a dependency?